### DISCUSSION TITLE
Subfolders in bookmarks

### DISCUSSION THREAD

**Discussion by yasio90** [organicmaps/organicmaps#780] [Ideas]
It would make me very happy if there would be possibility of having subfolders for bookmarks. I use bookmarks extensively and also download tracks from Wikiloc often - it makes the number of folders in bookmarks grow very quickly and currently it's simply annoying to browse them. A possibility of grouping bookmark folders in folders would solve this issue.
Just to give you an example: I followed a multi-day hike and for each day I downloaded a separate track (roughly 15 of them). Currently they form 15 separate folders - it would be much easier if I could have them in one folder.

**Comment by biodranik** [comment]
The idea is good, the only question is how to make it simple and convenient for everyone. A phone is not a desktop, and there is no mouse.

**Comment by Raphael0** [comment]
Hi, just to say that I would also love to see this feature, and I don't think that it should add too much complexity. The UI could be relatively straight forward with a link to add a folder at the very bottom under "create new list"

<img src="https://user-images.githubusercontent.com/23408445/124518161-0cafda80-dde6-11eb-9932-ea3606c178b7.jpeg" width="300" height="324" />

The new folder will be added to the other bookmark lists above (also with a checkbox to directly show/hide all nested bookmark lists) and tapping it will show a similar page with a list of the bookmark lists inside and a back button.

In my case, I have dozens of lists and I would love to be able to group them by theme: a "trips" folder in which I have a separate list for each trip; a "cities" folder; "projects" etc.

Today I have to scroll for a long time to find the correct list, and not being able to change their order makes it even more of a mess (just being able to arrange them would also be a big plus, I think this was also mentioned in another discussion)

**Reply by yasio90** [reply]
I agree that it could be something rather simple, @Raphael0's suggestion would work very well. A good example how it could work is how books are organized into collections on Amazon Kindle. One can add collections (folders) and from within the collections select all the books that are supposed be in this collection (by checking tick boxes). Or from the level of a book (by clicking the three dots) one can add the book to a collection. On Amazon Kindle one book can be in many collections, but I don't think it would be necessary in this case. 

Another possibility would be to have tags that could be added to bookmark folders. And user would select which tags should be displayed and which not.
